lib/Dredd.js

```diff
@@ -5,28 +5,26 @@ const fs = require('fs');
 const path = require('path');
 const request = require('request');
 const url = require('url');
+const parse = require('dredd-transactions/parse');
+const compile = require('dredd-transactions/compile');
 
-const dreddTransactions = require('dredd-transactions');
 const configureReporters = require('./configureReporters');
 const handleRuntimeProblems = require('./handleRuntimeProblems');
 const logger = require('./logger');
 const Runner = require('./TransactionRunner');
 const { applyConfiguration } = require('./configuration');
-
 const options = require('./options.json');
 
+
 const PROXY_ENV_VARIABLES = ['HTTP_PROXY', 'HTTPS_PROXY', 'NO_PROXY'];
 const FILE_DOWNLOAD_TIMEOUT = 5000;
 
-function removeDuplicates(arr) {
-  return arr.reduce((alreadyProcessed, currentItem) => {
-    if (alreadyProcessed.indexOf(currentItem) === -1) {
-      return alreadyProcessed.concat(currentItem);
-    }
-    return alreadyProcessed;
-  }, []);
+
+function unique(array) {
+  return Array.from(new Set(array));
```

Comment:
This function is intentionally here and in the `configuration.js` as these files are to be further refactored and I don't want to introduce more dependencies between them or any "utility functions" file just because of this.